### PR TITLE
feat: use provenance in JSR score

### PR DESCRIPTION
**Pull request by littledivy**
Closes https://github.com/jsr-io/jsr/issues/84

### CODE REVIEW

**Review by lucacasonato — COMMENTED** [review]
@littledivy Do you want this to be an "additional point" or update the maximum total by one?

**Comment by littledivy** [comment]
I think we update max total by one. 

oh i see, i have to bump the `MAX_SCORE` for that

**Comment by lucacasonato** [comment]
You also have to bump it on the JS side

**Comment by littledivy** [comment]
Can you point me to where that is?

File: frontend/routes/package/score.tsx [hunk 1]
```diff
@@ -164,6 +164,14 @@ export default function Score(
             This package is compatible with more than one runtime, and is marked
             as such in the package settings.
           </ScoreItem>
+          <ScoreItem
+            value={data.score.hasProvenance}
+            scoreValue={1}
+            title="Has provenance"
+          >
+            This package is published from a verifiable CI/CD workflow, and has
+            a public transparency log.
```
Comment (lucacasonato) [inline frontend/routes/package/score.tsx:173]: Add links to docs

**Review by lucacasonato — APPROVED** [review]
Oh my bad - that does not exist anymore. Then, LGTM modulo the docs nit!

File: frontend/routes/package/score.tsx (hunk 1 above)
Comment (lucacasonato) [reply frontend/routes/package/score.tsx]: Added a link, and updated the max down to 17. Let's see if this is widely adoptable, and once it is, we can tweak the max score :)

**Pull request merged**